build/scripts/generate_devworkspace_templates.sh

```diff
@@ -18,8 +18,11 @@ do
   if [ -n "$devfile_url" ]; then
     devfile_url=${devfile_url##*v2: }
     devfile_url=${devfile_url%/}
-    devfile_repo=${devfile_url%/tree*}
-    name=$(basename "${devfile_repo}")
+    npm_config_yes=true npx @eclipse-che/che-devworkspace-generator@${CHE_DEVWORKSPACE_GENERATOR_VERSION} \
```

Comment:
could you please add a comment why do we need to generate temp.yaml file for each sample

Reply:
done